_Shared/src/main/java/com/jagrosh/jdautilities/command/annotation/JDACommand.java

```diff
@@ -115,11 +115,6 @@
     /**
      * The {@link JDACommand.Cooldown JDACommand.Cooldown} for the command.
      *
-     * <p>This holds both metadata for both the
-     * {@link com.jagrosh.jdautilities.command.Command#cooldown Command#cooldown}
-     * and {@link com.jagrosh.jdautilities.command.Command#cooldownScope
-     * Command#cooldownScope}.
-     *
```

Comment:
IDEA was screaming at me because Command#cooldown wasn't supposed to be accessable.